src/components/ComboButton/__tests__/ComboButton.spec.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @file Combo button tests.
+ * @copyright IBM Security 2019
+ */
+
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import ComboButton from '../';
+
+import actions from '../_mocks_';
+
+describe('ComboButton', () => {
```

Comment:
It would be good to add tests to ensure that one action does not render the overflow menu